recipes/timechart/meta.yaml

```diff
@@ -0,0 +1,49 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "timechart" %}
+{% set version = "1.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 166a6d96110af04f2048573a6cff99f81fd4657c6b334d074e29e1c7ea995b83
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - pydm >=1.6.0
```

Comment:
Is this needed at pip install time?

Reply:
Addressed via https://github.com/conda-forge/staged-recipes/pull/11065/commits/48331451a38ce7e3cb5a259b6583f140e08379e1